src/app/proposals/[id]/page.tsx

```diff
@@ -544,14 +555,44 @@ const CalldataDisplay = (props: DecodedData) => {
                 <Typography tagVariant="span" className="font-semibold text-[16px] text-left">
                   {formattedInputName}
                 </Typography>
-                {InputComponent && (
-                  <InputComponent
-                    value={inputValue}
-                    htmlProps={{
-                      className: 'font-normal text-right',
-                    }}
-                  />
-                )}
+                <div className="flex flex-col items-end">
+                  {InputComponent && (
+                    <InputComponent
+                      value={inputValue}
+                      htmlProps={{
+                        className: 'font-normal text-right',
+                      }}
+                    />
+                  )}
+                  {formattedInputName === 'amount' && currentTokenSymbol && (
+                    <Span className="text-xs text-gray-400 mt-1">
+                      {(() => {
+                        // Check if prices exists
+                        if (!prices) return '≈ Price unavailable'
+
+                        const tokenSymbol = CURRENCY_ADDRESS_SYMBOLS_PAIR[currentTokenSymbol.toLowerCase()]
+
+                        // Safely check if the token exists in prices
+                        const tokenPrice = (prices as any)[tokenSymbol]
+
+                        // Check if we have a valid price
+                        if (tokenPrice && typeof tokenPrice.price === 'number') {
+                          let newPrice = getCombinedFiatAmount([
+                            {
+                              value: inputValue as bigint,
+                              price: tokenPrice.price,
+                              symbol: tokenSymbol,
+                              currency: 'USD',
+                            },
+                          ])
+                          return `≈ ${formatCurrency(newPrice.toNumber())} USD`
```

Comment:
Replace the approximation symbol to equal symbol. And move USD to the begining (ex: "= USD $1.00").

Reply:
Done